src/control/security/grpc_certs.go

```diff
@@ -0,0 +1,85 @@
+package security
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+
+	"google.golang.org/grpc/credentials"
+)
+
+func LoadServerCreds(caRootPath, certPath, keyPath string) (credentials.TransportCredentials, error) {
+
+	caPEM, err := LoadPEMData(caRootPath, SafeCertPerm)
+	if err != nil {
+		return nil, fmt.Errorf("could not load caRoot: %s", err)
```

Comment:
Why fmt.Errorf() instead of errors.Wrapf()?

Reply:
Most likely because I wrote this before the discussion on wrapping errors. Shouldn't be hard to change.